sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubScope.cs

```diff
@@ -94,7 +108,28 @@ public async ValueTask DisposeAsync()
         }
 
         /// <summary>
-        ///   Performs the tasks needed to create a new Event Hub instance with the requested
+        ///   It returns an EventHubScope after instanciating a management client
+        ///   and querying the consumer groups from the portal.
+        /// </summary>
+        ///
+        /// <returns>The <see cref="EventHubScope" /> that will be used in a given test run.</returns>
+        ///
+        private static async Task<EventHubScope> PopulateAsync()
```

Comment:
nit:  Can we consider renaming to something like `BuildScopeFromExistingResource` or similar?  The name was a bit confusing for me as I went through, since its situational to fleshing out a scope based on things that are not dynamically created.    With the name like `populate`, I expected it to be shared functionality across scenarios.